packages/graphiql-toolkit/src/create-fetcher/lib.ts

```diff
@@ -187,9 +210,21 @@ export async function getWsFetcher(
     return createWebsocketsFetcherFromClient(options.wsClient);
   }
   if (options.subscriptionUrl) {
+    const fetcherOptsHeaders = new Headers(fetcherOpts?.headers ?? {});
+    // @ts-expect-error: Current TS Config target does not support `Headers.entries()` method.
+    // However it is reported as "widely available" and so should be fine to use. This could
+    // would be more complicated without it.
+    // @see https://developer.mozilla.org/en-US/docs/Web/API/Headers/entries
+    const fetcherOptsHeadersEntries: [string, string][] = [
+      ...fetcherOptsHeaders.entries(),
+    ];
```

Comment:
Ideally this isn't needed. However I don't want to open a pandora's box by changing the `tsconfig.json` file.